docs/deploy-hyperlane.mdx

```diff
@@ -174,3 +174,10 @@ Finally, push local registry to github fork and submit a PR. Please include a [c
 
 Thank you for contributing to the future of permissionless interop 🫡
 :::
+
+:::tip
+
+**Next Steps**
+
+If you want deploy a Relayer to relay messages to your chain, continue on to [Deploy Hyperlane with Local Agents](/guides/deploy-hyperlane-local-agents.mdx#4-run-a-relayer)
```

Comment:
What's the ISM used in the core deployment? If it's the trusted / noop ISM a relayer is enough, but for a Multisig ISM a validator is needed as well

Reply:
It's just a trusted relayer